experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -3,6 +3,11 @@ Calling function: Process_Declaration
 Error message: Representation clause declaration
 Nkind: N_Attribute_Definition_Clause
 --
+Occurs: 88 times
```

Comment:
As this is adding unsupported feature ... was what we were doing before wrong or ... ?